# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/places/tests/favicons/test_moz-anno_favicon_mime_type.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c toolkit/components/places/tests/favicons/test_moz-anno_favicon_mime_type.js
--- a/toolkit/components/places/tests/favicons/test_moz-anno_favicon_mime_type.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/toolkit/components/places/tests/favicons/test_moz-anno_favicon_mime_type.js	Tue Feb 12 16:08:25 2019 +0000
@@ -39,7 +39,7 @@
     contentPolicyType: Ci.nsIContentPolicy.TYPE_INTERNAL_IMAGE_FAVICON,
   });
   let listener = new streamListener(PlacesUtils.favicons.defaultFaviconMimeType);
-  channel.asyncOpen2(listener);
+  channel.asyncOpen(listener);
   await listener.done.promise;
 });
 
@@ -51,7 +51,7 @@
     contentPolicyType: Ci.nsIContentPolicy.TYPE_INTERNAL_IMAGE_FAVICON,
   });
   let listener = new streamListener(PlacesUtils.favicons.defaultFaviconMimeType);
-  channel.asyncOpen2(listener);
+  channel.asyncOpen(listener);
   await listener.done.promise;
 });
 
@@ -71,6 +71,6 @@
     contentPolicyType: Ci.nsIContentPolicy.TYPE_INTERNAL_IMAGE_FAVICON,
   });
   let listener = new streamListener("image/png");
-  channel.asyncOpen2(listener);
+  channel.asyncOpen(listener);
   await listener.done.promise;
 });